src/java/org/apache/cassandra/metrics/CompactionMetrics.java

```diff
@@ -66,8 +94,9 @@
     /** Total number of compactions which have outright failed due to lack of disk space */
     public final Counter compactionsAborted;
 
-    /** The compaction strategy information for each table. */
-    public final Gauge<List<CompactionStrategyStatistics>> aggregateCompactions;
+    /** Total number of deleted expired SSTables */
+    public final Meter removedExpiredSSTables;
+    public final Meter deleteOnlyCompactions;
```

Comment:
Nit: small javadoc will be helpful.

Reply:
Fixed in b12567e0a160e562de7028b2fe1a38f6401ed4d2.